xmsannotator/tests/testthat/test_compute_rt_modules.R

```diff
@@ -0,0 +1,53 @@
+test_that("Integration test: RT based clustering works.", {
+    peaks <- readRDS("test-data/qc_solvent.rda")
```

Comment:
This file is not on github, a missing commit?

Reply:
See [3254247](https://github.com/RECETOX/recetox-xMSannotator/pull/17/commits/3254247fa31299013a2cd4228d371eb917456112)